cli/tools/lint/linter.rs

```diff
@@ -97,6 +99,15 @@ impl CliLinter {
     ext: Option<&str>,
   ) -> Result<(ParsedSource, Vec<LintDiagnostic>), AnyError> {
```

Comment:
The return type should be changed - if the file appears to be minified we won't have `ParsedSource`